aea/aea.py

```diff
@@ -98,6 +100,7 @@ def __init__(
             self.decision_maker.preferences,
             self.decision_maker.goal_pursuit_readiness,
             self.task_manager,
+            search_service_address,
```

Comment:
I am sorry, but I think we need to revert this. The AEA should not have to be aware of this value. We need to pass it to the agent context in a different manner. I will make a proposal ...

Reply:
removed